src/navigation/NavigationRoot.tsx

```diff
@@ -33,6 +35,7 @@ const NavigationRoot = () => {
             component={Onboarding}
             options={{ gestureEnabled: false }}
           />
+          {/* TODO: Handle when someone tries to link an account or sign a txn and they aren't authenticated in the app */}
```

Comment:
Good point, would this just go to some kind of error screen or maybe even just show a toast?

Reply:
should be handled now